packages/client/helpers/build.ts

```diff
@@ -6,15 +6,13 @@ import { build } from '../../../helpers/compile/build'
 import { fillPlugin } from '../../../helpers/compile/plugins/fill-plugin/fillPlugin'
 
 // we define the config for runtime
-const runtimeBuildConfig: BuildOptions = {
+const nodeRuntimeBuildConfig: BuildOptions = {
   name: 'runtime',
   entryPoints: ['src/runtime/index.ts'],
   outfile: 'runtime/index',
   bundle: true,
   define: {
-    'globalThis.NOT_PRISMA_DATA_PROXY': 'true',
-    // that fixes an issue with lz-string umd builds
-    'define.amd': 'false',
+    NOT_EDGE_CLIENT: 'true',
```

Comment:
Have been curious regarding this env var ever since I saw it under old name: why not the opposite, `EDGE_CLIENT: true`. Or, in this case, why not `NODE_CLIENT: true`. IMHO, it's much easier to mentally parse without double negatives.

Reply:
No actual reason. Fixed.